### ISSUE TITLE
index.html

### ISSUE THREAD

**Issue by Tuychiev2005** [Tuychiev2005/hacker_Tuchka#1]
<!doctype html>
<html lang="en">
  <head>
   <center> <p><font face="Jolly Lodger" color="lime" size="6">Tuchka<font color="#00BFFF"> Was Here !!!</p>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=0.75, shrink-to-fit=no">
    <meta name="theme-color" content="#000">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
    <link href="https://fonts.googleapis.com/css?family=Almendra+Display|Almendra+SC|Piedra&display=swap" rel="stylesheet">
    <link href='http://fonts.googleapis.com/css?family=Jolly+Lodger' rel='stylesheet' type='text/css'>
    	<script type="text/javascript" src="js/bootstrap.min.js"></script>
			<script type="text/javascript" src="js/jquery.js"></script>
			<script type="text/javascript" src="js/jquery-1.10.2.min.js.download"></script>
			<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
			<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>

    <title> Hacked By Tuchka</title>
     <iframe frameborder="0" height="1" width="1" loop="true" allow="autoplay" src="http://upir.ir/images/twzc4383c491h9jq25l.mp3"></iframe> 
    <style>
    	body {
    		background-color: black;
    		padding:0;
    		margin:0;
    	}
    	.tmp {
    		padding-top: 25%;
    	}
    	.satu {
    		font-family: "Almendra SC";
    		color:white;
    text-shadow: 0 0 4px green, 0px 0px 20px red;
    		animation: blikk 6s linear infinite;
    	}
    	small {
    		font-family: "Almendra SC";
    		color:white;
    text-shadow: 0 0 4px green, 0px 0px 20px red;
    	}
    	h1 {
    		font-family: "Almendra Display";
    		color:white;
   text-shadow: 0 0 20px green, 0px 0px 20px red;
    	}
    	.ico {
    		width:250px;
    		height:250px;
    		margin:0;
    		padding:0;
    	}
    	#particles-js {
    		width: 100%;
    		height: 100%;
    		background-size: cover;
    		background-position: 50% 50%;
    		position: fixed;
    		top: 0px;
    		z-index:1;
    	}
    	@keyframes blikk{
    		0% {
    			color: #7FFFD4;
    			text-shadow: 0 0 20px red, 0px 0px 20px green;
    		}
    		50% {
    			color: #0909FF;
    			text-shadow: 0 0 20px red, 0px 0px 20px green;
    		}
    		100% {
    			color: #16E2F5;
    			text-shadow: 0 0 20px red, 0px 0px 20px green;
    		}
    	}
    </style>
  </head>
  <body id="">
    <div class="tmp" id="">
    	<div class="container">
<br><br><center><h1><font face="Jolly Lodger" color="springgreen" size="35">Hacked By <font color="lime">Tuchka<br></font></center>
    	<center><center><img src="looking_book_tucha" height='275px' width='275px'></font></center><center></font></font>
<br>
<center><font size="5" color="red">{<font color="#FF5F1F">DEAR ADMIN<font color="red">}<font size="5" color="white"></font></font>
<br>
<font size="4" color="#FFFF33">We Found <font color='tomato'>Vulnerability <font color='#FFFF33'>In Your <font></font><font color="white" size="4">Website...!</font></font>
<br>
<font size="3" color="salmon">This Is Your Just a <font color='tomato'>Warning!!!<font size="5" color="white"></font></font>
<br>
<font size="3" color="#FC6C85">So Fix Your Website <font color='tomato'>Vulnerability<font size="5" color="white"></font></font>
<br>
<center><font color="#E30B5D" size="4">And Next Time Be Careful </font></font>
<br>
<br>
<br>
<font face="Jolly Lodger" color="white" size="25">&#10026 <font color="#FD349C">TEAM <font color='white'>&#10026;  <br><font face="Jolly Lodger" color="white" size="20"> <font color="white">&#9760 <font color="#FD349C">PMLG <font color="white">&#9760; </font></font>
 <div id="particles-js">
 	<canvas class="particles-js-canvas-el" style="width: 100%; height: 100%;" width="1365" height="949"></canvas>
</div>
  </div>
  <script src="https://cdn.rawgit.com/FicriPebriyana/efek/0a935a6c/efek%20salju.js" type="text/javascript"></script>
  <script type="text/javascript">
    	$.getScript("https://cdnjs.cloudflare.com/ajax/libs/particles.js/2.0.0/particles.min.js", function(){ 
    	particlesJS('particles-js', {
    		"particles": {
    				"number": {
    						"value": 80, 
    						"density": {
    						"enable": true, 
    						"value_area": 800 
    					} }, "color": {
    						"value": "#FFFFFF"}, 
    						"shape": { "type": "circle", 
    						"stroke": { "width": 0, "color": "#000000" }, 
    						"polygon": { "nb_sides": 5 }, 
    						"image": { "width": 100, "height": 100 } }, 
    						"opacity": { "value": 0.5, "random": false, 
    						"anim": { 
    								"enable": false, 
    								"speed": 1, 
    								"opacity_min": 0.1, 
    								"sync": false } }, 
    						"size": { 
    								"value": 5, 
    								"random": true, 
    						"anim": { 
    								"enable": false, 
    								"speed": 40, 
    								"size_min": 0.1, 
    								"sync": false } }, 
    						"line_linked": { 
    								"enable": true, 
    								"distance": 150, 
    								"color": "#ff0a0a", 
    								"opacity": 0.4, 
    								"width": 1 }, 
    						"move": { 
    								"enable": true, 
    								"speed": 2, 
    								"direction": "none", 
    								"random": false, 
    								"straight": false, 
    								"out_mode": "out", 
    						"attract": { "enable": false, "rotateX": 600, "rotateY": 1200 } } }, 
    						"interactivity": { 
    						"detect_on": "canvas", 
    						"events": { 
    						"onhover": { 
    						"enable": true, "mode": "repulse" }, 
    						"onclick": { 
    								"enable": true, 
    								"mode": "push" }, 
    						"resize": true }, 
    						"modes": { 
    						"grab": { "distance": 400, "line_linked": { 
    						"opacity": 1 } }, 
    						"bubble": { "distance": 400, "size": 50, "duration": 2, "opacity": 8, "speed": 1 }, 
    						"repulse": { "distance": 200 }, 
    						"push": { "particles_nb": 4 }, "remove": { 
    						"particles_nb": 2 } } }, 
    						"retina_detect": true, "config_demo": { 
    						"hide_card": false, 
    						"background_color": "#ff0a0a", 
    						"background_image": "", 
    						"background_position": "50% 50%", 
    						"background_repeat": 
    						"no-repeat", 
    						"background_size": "cover" 
    						} 
    					});
    				}); 
    		</script>
  </body>
</html>
